packages/actor-abstract-bindings-hash/lib/AbstractFilterHash.ts

```diff
@@ -33,7 +32,7 @@ export abstract class AbstractFilterHash<T extends Algebra.Operation> extends Ac
      * @return {boolean} If it exists.
      */
   public static doesHashAlgorithmExist(hashAlgorithm: string): boolean {
-    return getHashes().includes(hashAlgorithm);
+    return [ 'sha1' ].includes(hashAlgorithm);
```

Comment:
Should I keep using this form or should I move to something like `return 'sha1' === hashAlgorithm`? I guess it depends on how long you'd like to keep this module around and how much you'd like for others to feel like they can extend it by adding other hashing algorithms. Same applies for line 55, which could to with using the same style as this one regardless of which style you prefer.

Reply:
I'm ok with removing those functions, along with the parameters: https://github.com/comunica/comunica/blob/2d0818c64e5bfbbb334ecbccb7b5a98a69263d1c/packages/actor-abstract-bindings-hash/lib/AbstractFilterHash.ts#L17-L18

Reply:
Done!